### PR TITLE
memset the real size of res (Issue #25)

### PR DESCRIPTION
**Pull request by Tookmund**
I do not currently have the capability to test this, so it may not work. 
It is a pretty simple fix however.
#25

### CODE REVIEW

File: minix/commands/swifi/systest.c [hunk 1]
```diff
@@ -73,7 +73,7 @@ main(int argc, char * argv[])
     goto Cleanup;
   }
 
-  memset(res, 0, sizeof(res));
+  memset(res, 0, sizeof(*res));
```
Comment (dcvmoole) [inline minix/commands/swifi/systest.c]: I'm afraid that's not correct.. 'res' is a pointer, not an array, so you have to supply the size of what's being pointed to - in this case, "arg * sizeof(swifi_result_t)".

Comment (Tookmund) [inline minix/commands/swifi/systest.c]: That's what I get for not testing my changes... I think I've fixed it now.

**Comment by Tookmund** [comment]
having problems with rebase since its a branch created on github. ill work more on it later but for now I can't squash it sorry!

**Comment by Tookmund** [comment]
I'm not really sure what to do here. I cannot properly squash these commits for some reason.
Just use https://github.com/Stichting-MINIX-Research-Foundation/minix/pull/39.patch please with the commit message
`````
commands/swifi: fix incorrect memset size
````

**Comment by sambuc** [comment]
Hello @Tookmund,

I have submitted your change, after squashing it to a single commit, as well as editing the commit message. Is it OK for you as shown at:
    http://gerrit.minix3.org/2967

Regards,

**Comment by Tookmund** [comment]
That's  fine 
I forgot to add closes to the messages for these commits. Could you add closes #25 and closes #39 to the message?

**Comment by sambuc** [comment]
I don't think it is worth changing at that point, as it would retrigger the testing, which is almost done. 

I'll manually close the issues when it is merged, the `closes #` is a nice feature to save time, nothing more.

**Pull request closed without merging**